app/views/landmarks/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <h3><%= @landmark.name %></h3>
-<%= @landmark.user.email %>
+Creator: <%= @landmark.user.email %>
 <%= image_tag("#{@landmark.image}", height: '250', width: '250') %>
```

Comment:
Personally I prefer to style with CSS and not in the image tag, because it's counterintuitive to look here for information about how the image will be styled on the page.